README.md

```diff
@@ -134,6 +134,49 @@ becomes
       extra-lib-dirs:
         lib/darwin
 
+Multiple conditionals must fall under one `when`, for example:
+
+    when:
+      - condition: os(darwin)
+        extra-lib-dirs: lib/darwin
+      - condition: os(linux)
+        extra-lib-dirs: /usr/lib/x64
+
+becomes
+
+    if os(darwin)
+      extra-lib-dirs:
+        lib/darwin
+    if os(linux)
+      extra-lib-dirs:
+        /usr/lib/x64
+
+If conditionals are declared under different `when` clauses only the last one
+will make it to the `.cabal` file:
```

Comment:
@liskin Hey, interesting!  How about we don't emit warnings for unknown top-level field that starts with an underscore?  That way you can use fields that start with underscores for building kind of a `hpack` library.

I would certainly accept a patch.  So if you think thats the way to go, then feel free to open a PR!